pycontrails/models/accf.py

```diff
@@ -267,7 +267,7 @@ def eval(
         aCCFs, _ = clim_imp.get_xarray()
 
         # assign ACCF outputs to source
-        maCCFs = MetDataset(aCCFs)
+        maCCFs = MetDataset(aCCFs, sort=False)
```

Comment:
I wonder if you could achieve what you're doing here by manually making any tweaks to `aCCFs` first, then changing this line to `maCCFs = MetDataset(aCCFs, copy=False)`.